### DISCUSSION TITLE
Copy Button

### DISCUSSION THREAD

**Discussion by masondq** [arizon-dev/quickblaze-encrypt#13] [Ideas]
have a copy button, that copies the url once the encryption has occurred. or auto copy, on encryption.

**Comment by axtonprice** [comment]
I will implement this suggestion on the [dev](https://github.com/axtonprice-dev/quickblaze-encrypt/tree/dev) branch, and it will be merged to `main` once the next update is released.

**Comment by axtonprice** [comment]
⭐ This feature has been implemented to [v1.0.6-dev](https://github.com/axtonprice/quickblaze-encrypt/releases).
Thank you for your contribution @masondq.

![image](https://user-images.githubusercontent.com/37771600/163752555-66bf63ab-ae0f-41ee-8839-9b754536b790.png)
